src/OpenDebugAD7/AD7DebugSession.cs

```diff
@@ -879,7 +881,9 @@ protected override void HandleInitializeRequestAsync(IRequestResponder<Initializ
                 SupportsSetVariable = true,
                 SupportsFunctionBreakpoints = m_engineConfiguration.FunctionBP,
                 SupportsConditionalBreakpoints = m_engineConfiguration.ConditionalBP,
-                ExceptionBreakpointFilters = m_engineConfiguration.ExceptionSettings.ExceptionBreakpointFilters.Select(item => new ExceptionBreakpointsFilter() { Default = item.@default, Filter = item.filter, Label = item.label }).ToList(),
+                ExceptionBreakpointFilters = m_engineConfiguration.ExceptionSettings.ExceptionBreakpointFilters.Select(item => new ExceptionBreakpointsFilter() { Default = item.@default, Filter = item.filter, Label = item.label, SupportsCondition = item.supportsCondition, ConditionDescription = item.conditionDescription }).ToList(),
+                SupportsExceptionConditions = true,
```

Comment:
>SupportsExceptionConditions = true, [](http://example.com/codeflow?start=16&length=35)

You don't want `SupportsExceptionConditions`. That is a VS-only extension which is part of exception options.

```
        "supportsExceptionConditions": {
          "type": "boolean",
          "description": "The debug adapter supports using exception conditions in exception options."
        }
```